airmozilla/popcorn/views.py

```diff
@@ -92,7 +139,8 @@ def get(self, request, slug, form=None, conflict_errors=None):
             return self.cant_edit_event(event, request)
 
         context = {
-            'event': event
+            'event': event,
+            'slug': slug,
```

Comment:
The slug needs to exist within the HTML for the javascript to access which requires it to exist within the context.